hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/events/SCMEvents.java

```diff
@@ -196,6 +198,14 @@
   public static final TypedEvent<SafeModeStatus> SAFE_MODE_STATUS =
       new TypedEvent<>(SafeModeStatus.class);
 
+  /**
+   * This event is triggered by CommandStatusReportHandler whenever a
+   * status for CreatePipeline SCMCommand is received.
+   */
+  public static final TypedEvent<CreatePipelineStatus>
+      CREATE_PIPELINE_STATUS =
+      new TypedEvent<>(CreatePipelineStatus.class, "Create_Pipeline_Status");
+
```

Comment:
we don't need a destroyPipeline since it is not used any where ?

Reply:
You are right.  Close pipeline with timely a status confirm will make the propose more clear.  
Given this patch is already too big, I created HDDS-2176, Add new pipeline state “CLOSING” and new CLOSE_PIPELINE_STATUS command, to track the task.